sdk/redisenterprise/Azure.ResourceManager.RedisEnterprise/api/Azure.ResourceManager.RedisEnterprise.netstandard2.0.cs

```diff
@@ -159,11 +165,52 @@ public partial class RedisEnterprisePrivateEndpointConnectionResource : Azure.Re
 }
 namespace Azure.ResourceManager.RedisEnterprise.Models
 {
+    public partial class Capability
+    {
+        internal Capability() { }
+        public string Name { get { throw null; } }
+        public bool? Value { get { throw null; } }
+    }
+    public partial class ClusterPropertiesEncryptionCustomerManagedKeyEncryption
+    {
+        public ClusterPropertiesEncryptionCustomerManagedKeyEncryption() { }
+        public Azure.ResourceManager.RedisEnterprise.Models.ClusterPropertiesEncryptionCustomerManagedKeyEncryptionKeyIdentity KeyEncryptionKeyIdentity { get { throw null; } set { } }
+        public System.Uri KeyEncryptionKeyUri { get { throw null; } set { } }
+    }
+    public partial class ClusterPropertiesEncryptionCustomerManagedKeyEncryptionKeyIdentity
```

Comment:
Same here, could we rename this to `RedisEnterpriseCustomerManagedKeyEncryptionKeyIdentity`?

Reply:
renamed